
From: Gerd Knorr <kraxel@bytesex.org>

From: Paul Focke <paul.focke@pandora.be>

I recently upgraded from 2.4 to kernel 2.6 & noticed that the zoltrix radio
driver was not working.  Seems like a little typo.  I tested this on my system
and it's working fine now.  I doubt there are any other linux users in the
world who still use this card ;-)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/radio/radio-zoltrix.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/media/radio/radio-zoltrix.c~v4l-radio-zoltrix-fix drivers/media/radio/radio-zoltrix.c
--- 25/drivers/media/radio/radio-zoltrix.c~v4l-radio-zoltrix-fix	2004-06-19 13:58:52.166887304 -0700
+++ 25-akpm/drivers/media/radio/radio-zoltrix.c	2004-06-19 13:58:52.169886848 -0700
@@ -273,7 +273,7 @@ static int zol_do_ioctl(struct inode *in
 	case VIDIOCGAUDIO:
 		{
 			struct video_audio *v = arg;
-			memset(&v, 0, sizeof(*v));
+			memset(v, 0, sizeof(*v));
 			v->flags |= VIDEO_AUDIO_MUTABLE | VIDEO_AUDIO_VOLUME;
 			v->mode |= zol_is_stereo(zol)
 				? VIDEO_SOUND_STEREO : VIDEO_SOUND_MONO;
_
